default/widgets.py

```diff
@@ -0,0 +1,6 @@
+from django.forms.widgets import Widget
+
+
+class Display(Widget):
+    def render(self, name, value=None, attrs=None):
+        return '<span>{}</span>'.format(value)
```

Comment:
A non editable input is not very friendly looking. The color is mid grey and it does not look appealing. I really think a text is more convenient.
If it was out of Django I would just write it in the template, but here it's a widget to stick with the framework mindset.